packages/editor/src/components/autosave-monitor/index.js

```diff
@@ -53,6 +73,7 @@ export default compose( [
 			isDirty: isEditedPostDirty(),
 			isAutosaveable: isEditedPostAutosaveable(),
```

Comment:
It definitely does feel like this selector is useless or doesn't have the same meaning as its name.